### ISSUE TITLE
Env Dashboard Should filter the record as per folder

### ISSUE THREAD

**Issue by jeebitesh** [vipinsthename/environment-dashboard#18]
As we are using folder plugin and restrictive access to the various project.

This should support -> which should enable us to display only those jobs or values which are related to folder\view in jenkins.

I can also contribute.

This Plugin is very useful :)

**Comment by emoshaya-cognito** [comment]
yeah I'm facing the same issue in that the link to the job cannot be displayed
